src/main/java/com/duke/bot/DialogBox.java

```diff
@@ -0,0 +1,26 @@
+package com.duke.bot;
+
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+public class DialogBox extends HBox {
+
+    private Label text;
+    private ImageView displayPicture;
+
+    public DialogBox(Label l, ImageView iv) {
+        this.text = l;
+        this.displayPicture = iv;
```

Comment:
Now, the parameters in this method is clearly differed with the text and displayPicture values, so you don't need to add "this." in front of them. It indicates a value with the indicator belongs to not local but class.